docs/references/skyh5_memo.tex

```diff
@@ -0,0 +1,221 @@
+in units compatible with Jy or K sr. Some additional parameters are required depending on the component type. (\textit{component\_type})
+
+\item \textbf{Ncomponents}: \textit{int} The number of components in the sky model. This can be the number of individual
+compact sources, or it can include components of extended sources, or the number of pixels in a map. (\textit{Ncomponents})
+
+\item \textbf{spectral\_type}: \textit{string} This describes the type of spectral model for the components. The options are:
+``spectral\_index'', ``subband'', ``flat'', or ``full''. If the spectral model uses a spectral index, the reference\_frequency and
+spectral\_index parameters are required. The convention for the spectral index is $I=I_0 \frac{f}{f_0}^{\alpha}$, where
+$I_0$ is the stokes parameter at the reference\_frequency parameter $f_0$ and $\alpha$ is the spectral\_index parameter.
+Note that the spectral index is assumed to apply in the units of the stokes parameter (i.e. there is no additive factor of 2 applied
+to convert between temperature and flux density units).
+The subband spectral model is used for catalogs with multiple flux measurements at different frequencies (i.e. GLEAM
+\url{https://www.mwatelescope.org/science/galactic-science/gleam/}). For subband  spectral models, the freq\_array
+and freq\_edge\_array parameters are required to give the nominal (usually the central) frequency and the top and bottom of
+each subband respectively.
+The flat spectral model assumes no spectral flux dependence, which can be useful for testing.
+The full spectral model is used for catalogs with flux values at multiple frequencies that are not expected to have flux correlations as a function of frequency, so cannot not be interpolated to frequencies not included in the catalog. This is a good representation for e.g. Epoch of Reionization signal cubes.
+For full  spectral models, the freq\_array parameter is required to give the frequencies.
+(\textit{spectral\_type})
+
+\item \textbf{Nfreqs}: \textit{int}
+Number of frequencies if spectral\_type is ``full'' or ``subband'', 1 otherwise. (\textit{Nfreqs})
+astropy SkyCoord
+\item \textbf{history}: \textit{string} The history of the catalog. (\textit{history})
+\end{itemize}
+
+
+\subsection{Optional Parameters}
+\label{sec:opt_params}
+\begin{itemize}
+\item \textbf{name}: \textit{string} The name for each component. This is a one-dimensional array of size (Ncomponents).
+Note this is \textbf{required} if the component\_type is ``point''. (\textit{name})
+
+\item \textbf{skycoord}:
+A nested dataset that contains the information to create an \texttt{astropy SkyCoord} object representing the component positions.
+Note this is \textbf{required} if the component\_type is ``point''. The keys must include:
+	\begin{itemize}
+	\item \textbf{frame}: \textit{string} The name of the coordinate frame (e.g. ``icrs'', ``fk5'', ``galactic''). Must be a frame supported by \texttt{astropy}.
```

Comment:
I guess it's implicit in the fact that only Representation component names is listed as size (Ncomponents), but maybe be explicit that the skycoord entry only takes one frame, i.e., you can't have multiple components with different frames?